src/solvers/dg.jl

```diff
@@ -275,7 +286,8 @@ struct VolumeIntegralPureLGLFiniteVolumeO2{RealT <: Real, Basis, VolumeFluxFV,
     slope_limiter::Limiter # which type of slope limiter function
 end
 
-function VolumeIntegralPureLGLFiniteVolumeO2(basis::Basis, volume_flux_fv;
+function VolumeIntegralPureLGLFiniteVolumeO2(basis::Basis,
+                                             volume_flux_fv = flux_lax_friedrichs;
```

Comment:
Do you want to keep it as an optional position argument and not add it as a keyword argument?

Reply:
Hm good point. Could be made keyword argument, this would then result in a breaking change though.